inspirehep/modules/theme/static/js/detailed_record_init.js

```diff
@@ -22,11 +22,11 @@
 
 require([
   "js/plots",
-  "js/citation_modal",
-  "js/datatables"
+  // "js/citation_modal",
```

Comment:
Can you create another issue to make the citation modal and datatables work inside the detailed view?